lib/TPP/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ add_mlir_library(MLIRTPP
     ConvertCheckToLoops.cpp
     ConvertPerfToLoops.cpp
     ConvertPerfToFunc.cpp
-
+    ConvertTppToIdentity.cpp
```

Comment:
This isn't converting "TPP to Identity", it's converting `tensor.pack` and `tensor.unpack` to TPP identity. I'd name it `ConvertPackToXSMM.cpp` because we'll soon not be using TPP ops and will convert straight to an XSMM unary (with different flags for transpose and vnni).